tests/fixtures/network/l2_bridge.py

```
@@ -10,6 +11,11 @@
 from utilities.constants.networking import LINUX_BRIDGE
 
 
+@pytest.fixture(scope="class")
+def bridge_device_type(request: pytest.FixtureRequest) -> Any:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `bridge_device_type`

(ANN401)